lib/transactions.js

```diff
@@ -172,6 +172,14 @@ var generateOutputTransactions = function(poolRecipient, recipients, rpcData){
         poolRecipient
     ]));
 
+    if (rpcData.default_witness_commitment !== undefined){
```

Comment:
If there is no "default_witness_commitment" in the GBT response the witness commitment is not inserted.